source/overloading.tex

```diff
@@ -1937,59 +1937,27 @@
 For the purpose of ranking implicit conversion sequences as described
 in~\ref{over.ics.rank}, the ambiguous conversion sequence is treated
 as a user-defined conversion sequence that is indistinguishable from any
-other user-defined conversion sequence\footnote{The ambiguous conversion
-sequence is ranked with user-defined
-conversion sequences because multiple conversion sequences
-for an argument can exist only if they involve different
-user-defined conversions.  The ambiguous conversion sequence
-is indistinguishable from any other user-defined conversion
-sequence because it represents at least two user-defined conversion
-sequences, each with a different user-defined conversion, and
-any other user-defined conversion sequence must be
-indistinguishable from at least one of them.
-
-This rule prevents a function from becoming non-viable because of an ambiguous
-conversion sequence for one of its parameters.
-Consider this example,
-
-\begin{ttfamily}
-\begin{tabbing}
-\hspace{2in}\=\kill%
-\indent class B;\\
-\indent class A \{ A (B\&);\};\\
-\indent class B \{ operator A (); \};\\
-\indent class C \{ C (B\&); \};\\
-\indent void f(A) \{ \}\\
-\indent void f(C) \{ \}\\
-\indent B b;\\
-\indent f(b);\>\textrm{// ambiguous because \tcode{b} $\to$ \tcode{C} via constructor and}\\
-\indent \>\textrm{// \tcode{b} $\to$ \tcode{A} via constructor or conversion function.}
-\end{tabbing}
-\end{ttfamily}
-
-If it were not for this rule,
-\tcode{f(A)}
-would be eliminated as a viable
-function for the call
-\tcode{f(b)}
-causing overload resolution to select
-\tcode{f(C)}
-as the function to call even though it is not clearly the best
-choice.
-On the other hand, if an
-\tcode{f(B)}
-were to be declared then
-\tcode{f(b)}
-would resolve to that
-\tcode{f(B)}
-because the exact match
-with
-\tcode{f(B)}
-is better than any of the sequences required to match
-\tcode{f(A)}.}.
+other user-defined conversion sequence.  \begin{note}
```

Comment:
The note + example definitely belong to the "ambiguous conversion sequence" paragraph. The example highlights the entire preceding paragraph, not just the note, so nesting is not an option.

I could make the example (but not the note) a new paragraph. It seems we recently gravitate towards making long code examples separate paragraphs. Done.